addons/core/translations/resources/en-us.yaml

```diff
@@ -299,6 +299,8 @@ session:
   proxy-url:
     address-port: Address & port
     ssh: SSH
+  #  shell: Shell {shellCount, plural, one {} other {#}}
```

Comment:
Do we need to keep this translation still?

Reply:
No, this was pending on a discussion on whether we're going to move forward with multiple terminals